src/Core/Constants.cs

```diff
@@ -25,6 +25,7 @@ public static class Constants
         public static string iOSExtensionBiometricIntegrityKey = "iOSExtensionBiometricIntegrityState";
         public static string iOSShareExtensionClearCiphersCacheKey = "iOSShareExtensionClearCiphersCache";
         public static string iOSShareExtensionBiometricIntegrityKey = "iOSShareExtensionBiometricIntegrityState";
+        public static string iOSExtensionActiveUserIdKey = "iOSExtensionActiveUserId";
```

Comment:
I would also add `Constants.iOSExtensionActiveUserIdKey` to `MobileStorageService._preferenceStorageKeys`  to store this in the native prefs storage instead of the main app DB.